packages/data-explorer/src/components/utils/EndOfResults.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="alert alert-warning" role="alert">
+    <strong>Note:</strong> Displaying the maximum of {{dataDisplayLimit}} results, please use filtering to find your data.
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+
+export default Vue.extend({
+  name: 'EndOfResults',
+  props: {
+    dataDisplayLimit: {
+      type: Number,
+      required: true
+    }
+  }
+})
```

Comment:
suggest to move to domain specific folder ( for example , dataview or something like that) instead of generic 'util' folder. If it is truly a global component in the context of the app would suggest to just place it in the components folder.